codeToggles.js

```diff
@@ -1,13 +1,21 @@
 // This file is a map of github issue numbers with booleans
 // TODO: move to a modifiable location such as S3 or DynamoDB
 const toggles = {
+  6868: true,
   6916: true,
   6921: true,
   6929: true,
+  6933: true,
```

Comment:
This is the issue that should be set to false, right? Unless it does get done before this PR is merged, then disregard.

Reply:
6933 was marked done before this PR was created.

Reply:
Does this mean it was QA'ed? I didn't see that it was on the GitHub issue.

Reply:
6933 has been verified on flexion prod. I will close the issue now.